src/components/Review.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import { getReviews } from '../utils/utils';
+import { Link, useParams } from 'react-router-dom';
+import { getReviewById, getComments } from '../utils/utils';
+
+
+
+
+function Review() {
+    const [loading, setLoading] = useState(true);
+    const { review_id } = useParams();
+    const [comments, setComments] = useState(
+        [
+            {
+                "comment_id": 41,
+                "body": "Ex id ipsum dolore non cillum anim sint duis nisi anim deserunt nisi minim. Fugiat sint et proident ex do consequat est. Nisi minim laboris mollit cupidatat.",
+                "review_id": 18,
+                "author": "jessjelly",
+                "votes": 9,
+                "created_at": "2021-03-27T14:15:31.110Z"
+                }
+        ]
+
+    )
+ function deleteComments(id){
```

Comment:
I see that this is a placeholder but no need for it at this stage. We can always add the delete function when needed